pkgs/snaplet/registry.yaml

```diff
@@ -0,0 +1,25 @@
+packages:
+  - name: snaplet
+    type: http
+    description: Seed your PostgreSQL database with production-like data so you can code, debug, and test with ease.
+    link: https://www.snaplet.dev/
+    url: https://snaplet-public.s3-accelerate.amazonaws.com/cli/beta/snaplet-{{.OS}}-{{.Arch}}-{{.Version}}
+    overrides:
+      - goos: windows
+        url: https://snaplet-public.s3-accelerate.amazonaws.com/cli/beta/snaplet-{{.OS}}-{{.Arch}}-{{.Version}}.exe
+      - goos: darwin
+        goarch: arm64
+        replacements:
+          arm64: x64
+    supported_envs:
+      - linux
+      - darwin
+      - amd64
+      - windows
```

Comment:
```suggestion
      - windows/amd64
```

Reply:
Did you mean to remove `amd64` support? I believe that should still be supported, or am I missing something?

```suggestion
      - amd64
      - windows/amd64
```

Reply:
The keyword `darwin` includes `darwin/amd64`, so we don't need to specify `amd64`.